go.mod

```diff
@@ -1,21 +1,36 @@
-module github.com/J-A-M-P-S/go-etcstratum
+module github.com/dominant-strategies/go-quai-stratum
 
-go 1.16
+go 1.19
+
+replace github.com/dominant-strategies/go-quai => ../../djadih/go-quai
+
+replace github.com/dominant-strategies/go-quai-stratum => ../../djadih/go-quai-stratum
 
 require (
+	github.com/INFURA/go-ethlibs v0.0.0-20230210163729-fc6ca4235802
 	github.com/J-A-M-P-S/structs v1.1.0
-	github.com/btcsuite/btcd v0.21.0-beta // indirect
-	github.com/davecgh/go-spew v1.1.1
-	github.com/etclabscore/go-etchash v0.0.0-20210517131846-9a3cc202249e
+	github.com/dominant-strategies/go-quai v0.2.0-rc.0
 	github.com/ethereum/go-ethereum v1.10.3
-	github.com/garyburd/redigo v1.6.2 // indirect
 	github.com/gorilla/mux v1.8.0
 	github.com/robfig/cron v1.2.0
-	github.com/yvasiyarov/go-metrics v0.0.0-20150112132944-c25f46c4b940 // indirect
-	github.com/yvasiyarov/gorelic v0.0.7
-	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
-	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a // indirect
-	golang.org/x/sys v0.0.0-20210514084401-e8d321eab015 // indirect
-	gopkg.in/bsm/ratelimit.v1 v1.0.0-20160220154919-db14e161995a // indirect
 	gopkg.in/redis.v3 v3.6.4
 )
+
+require (
```

Comment:
why do we need 2 `require` ?

Reply:
https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17/69433550

I can combine the blocks if we think that's better. This is the result of running `go mod tidy`.

Reply:
Verbal acknowledgement (resolving)